pyproject.toml

```diff
@@ -13,34 +13,20 @@ include = ["data/mixnorm", 'data/mcfa']
 
 [tool.poetry.dependencies]
 python = ">=3.8"
```

Comment:
It seems like poetry would only install tensorflow (and everything after that) under Python 3.11 if I changed this to `>=3.9,<3.12`, unfortunately. Otherwise it was choosing tensorflow 2.11, which wasn't installable. The python upper bound might not be needed after the equivalent change to `space-rocks`.

Reply:
I don't quite understand what's going on, if you're on `python` 3.11 changing the `classy` `python` requirements shouldn't matter, right? All dependencies will be installed for `python` 3.11.

I will remove the upper bound from `rocks`. Either way, this line has to stay this way, these are the `python` versions `classy` has to support :-)

Reply:
Yes I’m not too happy with poetry’s limits either, but dependency management in python is overall just a mess. This looks good now, I’m merging it. Thanks Ari!